runway/hooks/staticsite/auth_at_edge/templates/shared.py

```diff
@@ -0,0 +1,351 @@
+# pylint: skip-file
+"""Shared functionality for the Auth@Edge Lambda suite.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+
+import base64
+import json
+import logging
+import re
+import time
+import traceback
+from datetime import datetime
+from random import random
+from urllib import request  # pylint: disable=no-name-in-module
+from urllib.parse import urlencode  # pylint: disable=no-name-in-module,import-error
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_config():
+    """Retrieve the configuration variables for the Auth@Edge suite.
+
+    Lambda@Edge restricts the ability to use environment variables. This configuration
+    object is generated with hard coded values via Runway.
+    """
+    # pylint: disable=undefined-variable
+    config = {{{DynamicConfig}}}  # noqa
```

Comment:
We should change the pre-formatted string to something _like_ `config = {'user_pool_id': 'tbd', 'http_headers': {}, 'token_issuer': 'tbd'}`, so the rest of the `unsupported-assignment-operation` pylint disabling lines can be removed.

Reply:
How would we swap the value then?

Reply:
I think it'd be fine to just wrap a substitution regex around the whole line (or nearly the whole line).

Reply:
Updated, take a peek and see if it matches expectations